sherpa/utils/__init__.py

```diff
@@ -1786,13 +1786,19 @@ def get_error_estimates(x, sorted=False):
     return (median, lval, hval)
 
 
-def poisson_noise(x):
+def poisson_noise(x, rng=None):
     """Draw samples from a Poisson distribution.
 
+    .. versionchanged:: 4.15.1
```

Comment:
```suggestion
    .. versionchanged:: 4.16.0
```

Reply:
This code conflicts with #1735 (although it's actually doing the same thing, mechanically it will lead to a conflcit).